### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Chiragmodi01**
## pr01: Homepage

- Created Top and Side Navbar
- Hamburger menu.

_site is not responsive_

**preview:**
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@import url('./styles/utils/colors.css');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```
Comment (aurobindoGupta) [inline src/App.css:2]: too many font weights stick to 4 max

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+.App {
+  text-align: center;
+  height: 100vh;
+  font-family: 'Roboto', sans-serif;
+}
+
+.App-logo {
+  height: 40vmin;
```
Comment (aurobindoGupta) [inline src/App.css:11]: is this a typo

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #22201c;
```
Comment (aurobindoGupta) [inline src/App.css:22]: variable

**Review by aurobindoGupta — APPROVED** [review]
nice

File: src/App.css (hunk 2 above)
Comment (Chiragmodi01) [reply src/App.css:11]: these are pre defined styles.

File: src/comps/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    
+  return (
+    <div className='Navbar'>
+        <div className="navbar_left">
+        <IoMdMenu size='1.5em' className='ham-menu_icon' onClick={() => setHideAside(prev => !prev)}/>
+        <GrDocumentNotes size='2em' className='notero-icon'/>
+        <h1 className="brandname">Notero</h1>
+        </div>
+        <div className="navbar_right">
+        <div className='searchbox'>
```
Comment (sashaboi) [inline src/comps/Navbar.jsx:18]: Only suggestion is the placeholder text for the search bar has bad contrast when it comes to green bg.
Could be color white for the placeholder property only, I tried to find more about this here :
https://www.w3schools.com/howto/howto_css_placeholder.asp

**Review by sashaboi — APPROVED** [review]
SOO good this UI, clean AF 💯 

Looking forward for more versions of this project.

**Review by sashaboi — COMMENTED** [review]
image
Favicon could look even more kick ass on the darker green shade maybe, for a better contrast ratio?

File: src/comps/Navbar.jsx (hunk 4 above)
Comment (Chiragmodi01) [reply src/comps/Navbar.jsx:18]: Good point, my bad i haven't noticed!

**Comment by Chiragmodi01** [comment]
> image Favicon could look even more kick ass on the darker green shade maybe, for a better contrast ratio?

Prefered in accent color, but will experiment more!

**Pull request merged**